packages/editor-ui/src/components/DebugPaywallModal.vue

```diff
@@ -16,7 +16,7 @@ const i18n = useI18n();
 			<n8n-text>
 				{{ i18n.baseText('executionsList.debug.paywall.content') }}
 				<br />
-				<n8n-link :to="i18n.baseText('executionsList.debug.paywall.link.url')">
+				<n8n-link :to="i18n.baseText('executionsList.debug.paywall.link.url')" new-window>
```

Comment:
Why are we using i18n for a link?

Reply:
It's a good practice in case of multilanguage docs